### PR TITLE
Disable tests failing on NAOT

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Comment by jkotas** [comment]
Follow up on #101317. cc @jkoritzinsky

**Review by filipnavara — APPROVED** [review]

**Comment by jkotas** [comment]
@jkoritzinsky I am getting the following error:

`Generator 'XUnitWrapperGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'NullReferenceException' with message 'Object reference not set to an instance of an object.'`

It seems to be caused by https://github.com/dotnet/runtime/pull/102065/files#diff-eb5ac758f23f3275cf69654218fd507c2fc49cb552c1885417c202da5fa18872R133 . Are `ActiveIssue` attributes in IL tests expected to work?

**Comment by jkotas** [comment]
The remaining failures are infra issues

**Review by jkoritzinsky — APPROVED** [review]

File: src/tests/JIT/Methodical/doublearray/dblarray4.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class DblArray4
     [Fact]
     [SkipOnCoreClr("This test is not compatible with GCStress.", RuntimeTestModes.AnyGCStress)]
     [OuterLoop]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/101284", typeof(TestLibrary.Utilities), nameof(TestLibrary.Utilities.IsNativeAot))]
```
Comment (tannergooding) [inline src/tests/JIT/Methodical/doublearray/dblarray4.cs:25]: This is failing in some outerloop jobs with
> src/tests/JIT/Methodical/doublearray/dblarray4.cs(25,76): error CS0246: The type or namespace name 'TestLibrary' could not be found (are you missing a using directive or an assembly reference?) [/__w/1/s/src/tests/JIT/Methodical/doublearray/dblarray4_cs_d.csproj]

For example, from https://dev.azure.com/dnceng-public/public/_build/results?buildId=673058&view=logs&jobId=3725811e-4a8b-5f6a-b03b-a353ee894148&j=3725811e-4a8b-5f6a-b03b-a353ee894148&t=d59fccfb-2b13-5841-75bf-bfec705ee16a

Comment (jkotas) [reply src/tests/JIT/Methodical/doublearray/dblarray4.cs:25]: #102134

**Pull request merged**